tipg/factory.py

```diff
@@ -1364,3 +1391,359 @@ async def tilematrixset(
             OGC Specification: http://docs.opengeospatial.org/per/19-069.html#_tilematrixset
             """
             return self.supported_tms.get(tileMatrixSetId)
+
+        # NOTE: The OGC Tiles API specification includes `/conformances` and `/` endpoints
+        # Those two endpoints are optional in the factory and only added if `full=True`.
+        if self.full:
+
+            @self.router.get(
+                "/conformance",
```

Comment:
`conformance` and `/` are OGC core specification so in theory we could create an OGC Tiles API with them + the tiles endpoints